manifest.json

```diff
@@ -275,7 +275,7 @@
     {
       "description": "Shows \"git blame\" information of a line *([screenshot](https://raw.githubusercontent.com/juliardi/lite-xl-gitblame/main/screenshot_1.png))*",
       "version": "0.1",
```

Comment:
Should probably bump this too.

Reply:
> I think you should add a `"replaces": [ "treeview-menu-extender" ]` field, so that people will automatically use the new version. @adamharrison is that the correct syntax/usage?

This is the correct syntax, yes.

Reply:
@Guldoman But there is no newer version for `gitblame` though.

Reply:
You changed the commit it was pointing to, so it should be bumped.

Reply:
alright, it's done. can you check it please?